hazelcast/serialization/portable/serializer.py

```diff
@@ -53,18 +53,18 @@ def find_portable_version(self, factory_id, class_id, portable):
         return current_version
 
     def create_new_portable_instance(self, factory_id, class_id):
-        try:
-            portable_factory = self._portable_factories[factory_id]
-        except KeyError:
+        portable_factory = self._portable_factories.get(factory_id)
+        if not portable_factory:
             raise HazelcastSerializationError(
                 "Could not find portable_factory for factory-id: %s" % factory_id
             )
 
-        portable = portable_factory[class_id]
-        if portable is None:
+        portable = portable_factory.get(class_id)
+        if not portable:
```

Comment:
Checking with `is None` looks more correct to me.

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/619/commits/83eeea0e7068e38f06f360531cb84d0108f4a5bb